input-forms.md

```diff
@@ -0,0 +1,82 @@
+# Input-forms Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+## Main Endpoint
+**/api/configuration/input-forms**   
+
+Provide access to the configured input-forms. It returns the list of existent input-forms.
+Please note that starting from DSpace 7 each input-form will describe a single page of inputs, combining different pages togheter is done with the [submission-definition](submission-panels.md), aka the item-submission.xml configuration file 
+
+Example: to be provided
+
+## Single Input-Form 
+**/api/configuration/input-forms/<:form-name>**
+
+Provide detailed information about a specific input-form. The JSON response document is as follow
+```json
+{
+  "name": "traditional",
```

Comment:
Above you state that each `input-form` will not represent a single *page*.  That said, this likely should not be named "traditional", as it almost seems to map to the ["traditional" form in input-forms.xml](https://github.com/DSpace/DSpace/blob/master/dspace/config/input-forms.xml#L39) which is a _multi-page form_.